python/src/trezorlib/debuglink.py

```diff
@@ -357,7 +400,7 @@ class TrezorClientDebugLink(TrezorClient):
     # without special DebugLink interface provided
     # by the device.
 
-    def __init__(self, transport, auto_interact=True):
+    def __init__(self, transport: "TransportType", auto_interact: bool = True) -> None:
```

Comment:
why is this not `Transport`?

Reply:
Same as above, because of the `find_debug` missing. Fixed in a896ca2b0